ofrak_core/ofrak/gui/server.py

```diff
@@ -503,6 +479,7 @@ def get_query_string_as_pjson(request: Request) -> Dict[str, PJSONType]:
         _host = "127.0.0.1"
         _port = 8080
 
+    backend: Optional[str]
     if len(sys.argv) == 4:
         backend = sys.argv[3]
     else:
```

Comment:
This is another nit I'm noticing not in your work Marc, but in the existing code you're working around. On line 505, when the "angr" backend is selected, we need to also import and discover `ofrak_capstone` like in the "binary-ninja" case. Otherwise, we won't get any instructions.